### PR TITLE
feat(admin-gui): admin service project gui (DSP-1738)

### PR DESCRIPTION
**Pull request by mdelez**
resolves DSP-1738

### CODE REVIEW

**Comment by mdelez** [comment]
@BalduinLandolt @subotic once you have an event store docker image up and running you can run `make admin-service-run` and view the GUI on http://localhost:8080/projects

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+/dasch-service-platform-repo.iml
+/modules.xml
+/vcs.xml
```
Comment (BalduinLandolt) [inline .idea/.gitignore:8]: do we want `.idea` stuff in the repo?

File: services/admin/backend/api/handler/project.go [hunk 2]
```diff
@@ -467,30 +466,39 @@ func listProjects(service project.UseCase) http.Handler {
...
+	//r.Handle("/v1/projects/{id}", n.With(
+	//	negroni.Wrap(deleteProject(service)),
+	//)).Methods("DELETE", "OPTIONS").Name("deleteProject")
 
-	r.Handle("/v1/projects/{id}", n.With(
-		negroni.Wrap(updateProject(service)),
-	)).Methods("PUT", "OPTIONS").Name("updateProject")
+	//r.Handle("/v1/projects", n.With(
+	//	negroni.Wrap(listProjects(service)),
+	//)).Methods("GET", "OPTIONS").Name("listProjects")
```
Comment (BalduinLandolt) [inline services/admin/backend/api/handler/project.go:493]: sure you want to keep those?

File: services/admin/backend/cmd/main.go [hunk 3]
```diff
@@ -71,47 +52,73 @@ func main() {
...
-	}
-	err = srv.ListenAndServe()
-	if err != nil {
-		log.Fatal(err.Error())
-	}
+	handler.MakeProjectHandlers(&s.Router, projectService)
+
+	log.Fatal(s.ListenAndServe())
+
+	// organizationRepository := repository.NewInmemDB()
```
Comment (BalduinLandolt) [inline services/admin/backend/cmd/main.go:63]: those commented out lines could be deleted too, right?

File: services/admin/frontend/Modal/Dialog.svelte [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+    input {
+        width: 100%;
+    }
+
+    .buttons {
+        display: flex;
+        justify-content: space-between;
+    }
+</style>
```
Comment (BalduinLandolt) [inline services/admin/frontend/Modal/Dialog.svelte:63]: in Svelte it seems to be more common to have `<style>` at the end of the component

**Review by BalduinLandolt — APPROVED** [review]
I didn't run it, just looked at the code. Looks good to me.

I added some comments, nothing important though.

File: .idea/.gitignore (hunk 1 above)
Comment (mdelez) [reply .idea/.gitignore:8]: idk really, this is part of my IDE battle 😓

File: services/admin/backend/api/handler/project.go (hunk 2 above)
Comment (mdelez) [reply services/admin/backend/api/handler/project.go]: nah, removed in b681dfc :)

File: services/admin/backend/cmd/main.go (hunk 3 above)
Comment (mdelez) [reply services/admin/backend/cmd/main.go]: removed in b681dfc

File: services/admin/frontend/Modal/Dialog.svelte (hunk 4 above)
Comment (mdelez) [reply services/admin/frontend/Modal/Dialog.svelte]: done in b681dfc

**Review by BalduinLandolt — APPROVED** [review]
looks good

File: .idea/.gitignore (hunk 1 above)
Comment (subotic) [reply .idea/.gitignore:8]: you should probably remove it. editor configurations don't belong in the repository.

File: public/BUILD.bazel [hunk 5]
```diff
@@ -2,9 +2,5 @@ package(default_visibility = ["//visibility:public"])
...
 filegroup(
     name = "public",
-    srcs = glob(["assets/**/*"]) + glob(["build/**/*"]) + [
-        "favicon.png",
-        "global.css",
-        "index.html",
-    ],
-)
+    srcs = glob(["assets/**/*"]) + glob(["build/**/*"])
+)
```
Comment (subotic) [inline public/BUILD.bazel:6]: can you please run `make buildifier` to clean up the build files.

File: public/admin/index.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+  <link rel='icon' type='image/png' href='favicon.png'>
+  <link rel='stylesheet' href='global.css'>
+  <link rel='stylesheet' href='/build/bundle.css'>
+
+  <script defer src='/build/bundle.js'></script>
+
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <!-- second script moved to separate file -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-9MD8RKZJHE"></script>
```
Comment (subotic) [inline public/admin/index.html:32]: we don't use google analytics anymore. can you please remove this.

**Review by subotic — APPROVED** [review]
LGTM, thanks! Congrats :-)

File: public/admin/index.html (hunk 6 above)
Comment (mdelez) [reply public/admin/index.html]: removed in a6606a5

File: public/BUILD.bazel (hunk 5 above)
Comment (mdelez) [reply public/BUILD.bazel]: done in a6606a5

File: .idea/.gitignore (hunk 1 above)
Comment (mdelez) [reply .idea/.gitignore]: removed in a6606a5

**Pull request merged**